apps/trackers/common.py

```diff
@@ -40,6 +40,13 @@ def tracker(self):
         """
         return self.instance
 
+    @property
+    def creation(self):
```

Comment:
How about `creating` or `is_creating`? IMO this way it's clearer that it's a boolean property.

Reply:
Makes sense. I will rename it

Reply:
The same naming was also used in the BBSync module and for the Bugzilla trackers. I changed the naming throughout to unify it on `is_creating`